operational/flexera/msp/customer_usage_insights_report/README.md

```diff
@@ -0,0 +1,57 @@
+# MSP Customer Usage Insights Report
+
+## What It Does
+
+This policy generates a comprehensive report on the usage of MSP Customer Organizations. It provides operational capabilities related to managing MSP Customer Organizations, including cost analysis and user activity metrics.
+
+## How It Works
+
+- The policy retrieves a list of MSP Customer Organizations.
+- It gathers cost data for each organization, broken down by billing source (e.g., AWS, Azure, GCP, Oracle).
+- It calculates user activity metrics, including the number of active and inactive users.
+- The policy generates a detailed markdown report summarizing the costs and user activity for each organization.
+- It estimates 12-month costs using the most recent 3 months of costs.
+- It identifies organizations with 100% inactive users and includes their total costs in the report.
+- It allows excluding organizations with total costs below a specified threshold.
+- It allows excluding certain costs from the net cost calculations and reports excluded costs separately.
+
+## Input Parameters
+
+This policy has the following input parameters required when launching the policy.
+
+- *Email Addresses* - Email addresses of the recipients you wish to notify when new incidents are created.
+- *Automatic Actions* - When this value is set, this policy will automatically take the selected actions.
+- *Allow/Deny Child Orgs* - Determines whether the Allow/Deny Child Orgs List parameter functions as an allow list (only providing results for the listed organizations) or a deny list (providing results for all organizations except for the listed organizations).
+- *Allow/Deny Child Orgs List* - A list of allowed or denied Child Organizations to include in the report.
+- *Prefix String* - Prefix to indicate the Group should be synced. This is the first part of the string that needs to be in the description.
+- *Excluded User Regex* - A list of email regex patterns to exclude from the active/inactive user counts. We generally recommend always including `.*@flexera.com` to exclude Flexera employees from the counts. Add any other regex patterns for "internal" users that should be excluded from the active/inactive user counts.
+- *Included User Regex* - A list of email regex patterns to include in the active/inactive user counts. This is useful for specifically including certain users, especially when using exclusions. Default is `.*` which includes all users not explicitly excluded.
+- *Active Days Threshold* - The number of days to consider a user as active. Default is 30 days.
+- *Processed Spend Threshold* - The minimum total processed spend for an organization to be included in the report. Default is -1 which disables the threshold and shows all Orgs.
+- *Billing Period* - The billing period to report on. Options are "Previous Month", "Previous 12 Months", or "Specific Month".
```

Comment:
Why these options only? Would it make sense to allow the user to specify a number of months back?